conftest.py

```diff
@@ -147,6 +147,25 @@ def default_configuration():
         enable_unsafe_features=True,
         use_insecure_key_cache=True,
         insecure_key_cache_location="ConcreteNumpyKeyCache",
+        fhe_simulation=False,  # Simulation compilation is done lazilly on circuit.simulate
```

Comment:
we try to avoid in-line comment so better to move it above please !

Reply:
sure. I moved it locally. Will push it if ever I have something important to push. Otherwise it will go in one of the other PR.

Reply:
Done in #562